app/helpers/application_helper.rb

```diff
@@ -9,10 +9,19 @@ def show_title?
     @show_title
   end
 
+  def uswds_class_for flash_type
```

Comment:
let's put `flash_type` inside parens since that is prettt common in ruby method definitions (although not required, ppl tend to do it)